test/plugin/test_exception_detector.rb

```diff
@@ -270,6 +270,222 @@ class ExceptionDetectorTest < Test::Unit::TestCase
+  printError( () { new NoType(); } );
+                       ^
+
+
+#0      _TypeError._throwNew (dart:core-patch/errors_patch.dart:82)
+#1      main.<anonymous closure> (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:7:24)
+#2      printError (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:36:13)
+#3      main (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:7:3)
+#4      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#5      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_FORMAT_A_ERR = <<END.freeze
+Unhandled exception:
+FormatException: format exception
+#0      main.<anonymous closure> (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:25:20)
+#1      printError (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:42:13)
+#2      main (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:25:3)
+#3      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_FORMAT_B_ERR = <<END.freeze
+Unhandled exception:
+FormatException: Invalid base64 data (at line 3, character 8)
+this is not valid
+       ^
+
+#0      main.<anonymous closure> (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:24:20)
+#1      printError (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:42:13)
+#2      main (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:24:3)
+#3      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_NO_METHOD_A_ERR = <<END.freeze
+Unhandled exception:
+NoSuchMethodError: No constructor 'TypeError' with matching arguments declared in class 'TypeError'.
+Receiver: Type: class 'TypeError'
+Tried calling: new TypeError("Invalid base64 data", "invalid", 36)
+Found: new TypeError()
+#0      NoSuchMethodError._throwNew (dart:core-patch/errors_patch.dart:196)
+#1      main.<anonymous closure> (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:8:39)
+#2      printError (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:36:13)
+#3      main (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:8:3)
+#4      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#5      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_NO_METHOD_B_ERR = <<END.freeze
```

Comment:
How about calling the above `DART_NO_METHOD_ERR` and this one `DART_NO_METHOD_GLOBAL_ERR`?